src/csvcubed/utils/sparql_handler/data_cube_state.py

```diff
@@ -125,33 +119,22 @@ def _detect_shape_for_cube(
             for (csv_url, measures_with_shape) in map_csv_url_to_measure_shape.items()
         }
 
-    """
-    Public getters for the cached properties.
-    """
-
-    def get_column_definitions_for_csv(self, csv_url: str) -> List[ColumnDefinition]:
-        """
-        Getter for _col_names_col_titles cached property.
-        """
-        result: List[ColumnDefinition] = self._get_value_for_key(
-            csv_url, self._column_definitions
-        )
-        return result
+    # Public getters for the cached properties.
 
     def get_unit_for_uri(self, uri: str) -> Optional[UnitResult]:
-        """ """
+        """The function returns a single unit from the uri if there is any."""
         return self._units.get(uri)
 
     def get_units(self) -> List[UnitResult]:
-        """ """
+        """This function gets the value of the unit and returns it as a list."""
```

Comment:
Yeah, it's just been moved into the `CsvWInspector` since it's useful in both `CodeListInspector` and `DataCubeInspector` and describes things at the CSV-W level.